internal/interceptor.go

```diff
@@ -416,6 +421,11 @@ type ClientUpdateWorkflowInput struct {
 	WaitForStage        WorkflowUpdateStage
 }
 
+type ClientUpdateWithStartWorkflowInput struct {
+	UpdateInput            *ClientUpdateWorkflowInput
+	StartWorkflowOperation WithStartWorkflowOperation
```

Comment:
May be worth noting that the interceptor can't do much with this. They can't even know what workflow it applies to. We don't have to do anything now, but I would expect at some point for interceptor authors to ask how to get details of the start the same way they can get details of the update.